browser/ui/webui/settings/brave_default_extensions_handler.cc

```diff
@@ -315,15 +305,6 @@ void BraveDefaultExtensionsHandler::IsWidevineEnabled(
 }
 
 void BraveDefaultExtensionsHandler::OnWalletTypeChanged() {
-  if (brave_wallet::GetDefaultEthereumWallet(profile_->GetPrefs()) ==
-      brave_wallet::mojom::DefaultWallet::CryptoWallets) {
-    return;
-  }
-  extensions::ExtensionService* service =
-      extensions::ExtensionSystem::Get(profile_)->extension_service();
-  service->DisableExtension(
-      kEthereumRemoteClientExtensionId,
-      extensions::disable_reason::DisableReason::DISABLE_USER_ACTION);
 }
 
```

Comment:
We now don't need subscription to `kDefaultEthereumWallet` and `OnWalletTypeChanged` handler